tests/Test/Hspec/Missing.hs

```diff
@@ -0,0 +1,16 @@
+{-# OPTIONS_GHC -Wall -Werror #-}
+
+{-# LANGUAGE OverloadedStrings #-}
+module Test.Hspec.Missing where
```

Comment:
could you add a haddock comment for the module that links to `docs/test-suites.md`?